templates/repo/activity.tmpl

```diff
@@ -6,12 +6,10 @@
 			<div class="ui right">
 				<!-- Period -->
 				<div class="ui floating dropdown jump filter">
-					<div class="ui basic compact button">
-						<span class="text">
-							{{.locale.Tr "repo.activity.period.filter_label"}} <strong>{{.PeriodText}}</strong>
-							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
-						</span>
-					</div>
+					<span class="ui basic compact button text">
```

Comment:
span or button?

Reply:
Oh .... it's inside a dropdown .... it shouldn't be a `<button>` 😂

Then I guess the `ui button` classes should be on the `ui dropdown` instead of an inner element, like others?

Either is fine, just curious.